pkgs/top-level/all-packages.nix

```diff
@@ -13891,9 +13891,10 @@ in
   ### DEVELOPMENT / PERL MODULES
 
   perlInterpreters = callPackages ../development/interpreters/perl {};
-  inherit (perlInterpreters) perl528 perldevel;
+  inherit (perlInterpreters) perl528 perl530 perldevel;
 
   perl528Packages = recurseIntoAttrs perl528.pkgs;
+  perl530Packages = recurseIntoAttrs perl530.pkgs;
```

Comment:
Do we want to recurse into both sets? They will both be build by Hydra and show up with `nix-env`.

Reply:
build by Hydra is desirable, it is going to be default ```pkgs.perl``` in about a month, so comparison of ```pkgs.perl528Packages``` and ```pkgs.perl530Packages``` would be useful for making decision about switching to ```perl530```.

About ```nix-env``` I have no opinion.